packages/grafana-ui/src/components/Menu/Menu.tsx

```diff
@@ -66,17 +66,17 @@ export const Menu = Object.assign(MenuComp, {
 const getStyles = (theme: GrafanaTheme2) => {
   return {
     header: css({
-      padding: `${theme.spacing(0.5, 1, 1, 1)}`,
+      padding: theme.spacing(0.5, 1, 1, 1),
     }),
     headerBorder: css({
       borderBottom: `1px solid ${theme.colors.border.weak}`,
     }),
     wrapper: css({
-      background: `${theme.colors.background.primary}`,
-      boxShadow: `${theme.shadows.z3}`,
-      display: `inline-block`,
-      borderRadius: `${theme.shape.radius.default}`,
-      padding: `${theme.spacing(0.5, 0)}`,
+      background: theme.colors.background.primary,
+      boxShadow: theme.shadows.z3,
+      display: 'inline-block',
+      borderRadius: theme.shape.radius.default,
+      padding: theme.spacing(0.5, 0),
```

Comment:
this also isn't really doing anything, it's just removing the unnecessary template string wrapping

Reply:
I think we could use `Box` and get rid of these styles altogether.